master/internal/job/job.go

```diff
@@ -223,8 +223,14 @@ func (j *Jobs) Receive(ctx *actor.Context) error {
 				return nil
 			}
 		}
-		if len(errors) > 0 {
-			ctx.Respond(fmt.Errorf("encountered the following errors: %s", strings.Join(errors, ", ")))
+		if len(errors) == 1 {
+			ctx.Respond(errors[0])
+		} else if len(errors) > 1 {
+			msgs := make([]string, 0)
```

Comment:
Why not make list a list of strings to begin with, and only convert to an error when we send? Then we wouldn't have to convert back and forth here.